### ISSUE TITLE
Chainlink Alerting: eth_balance metric is misleading. It refers to the specific evm network's balance and not the ETH balance

### ISSUE THREAD

**Issue by dillu24** [SimplyVC/panic#165]
Although misleading, the `eth_balance` metric retrieves the balance of the node on the specific network. For example for a BSC node, the `eth_balance` network returns the `BNB` balance.

As a result, we need to change every instance in our node and contracts monitors data transformers, data stores and alerters where we refer to the balance as the ethereum balance.

In addition to this, some alerts have the `ETH` moniker inside their description. To solve this it is suggested that we use this endpoint inside the monitors https://chainid.network/chains.json or https://chainid.network/chains_mini.json obtained from https://github.com/ethereum-lists/chains

To summarise, the following must be done:
- [x] Update parsing/keys and refactor ETH-related variables/functions in alerter
- [x] Update alert codes in API
- [x] Add logic for getting correct ticker using ChainID
- [x] Update documentation of said alerts
- [x] Upgradeability in the form of a script included in hotfix
- [x] Change alert keys, titles, and descriptions of ETH-related CL alerts in installer #166 
- [x] Release hotfix

**Comment by Cherrett** [comment]
We must also update the documentation of the modified alerts and update the eth related CL alerts within the installer (title, description, keys).

**Comment by Cherrett** [comment]
To summarise, the following must be done:

- Change title and descriptions of eth related CL alerts
- Store alert configs correctly using updated keys
- Update parsing from alerter
- Update documentation of said alerts.
- Upgradeability in the form of a script included in hotfix

**Comment by crsfalzon** [comment]
Hotfix with #106
